dist/jose-jwe.js

```diff
@@ -553,10 +562,10 @@ JoseJWE.prototype.encryptCek = function(key_promise, cek_promise) {
  */
 JoseJWE.prototype.encryptPlainText = function(cek_promise, plain_text) {
   // Create header
-  var jwe_protected_header = Utils.Base64Url.encode(JSON.stringify({
-    "alg": this.key_encryption.jwe_name,
-    "enc": this.content_encryption.jwe_name
-  }));
+  this.headers.alg = this.key_encryption.jwe_name;
```

Comment:
Given that it's async (promises), you probably don't want to reuse `JoseJWE` objects. Merging into a new dictionary seems reasonable.